app/src/main/res/layout/dialog_image_description.xml

```diff
@@ -11,28 +12,53 @@
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:layout_weight="1"
-        android:contentDescription="@string/post_media_image"/>
+        android:contentDescription="@string/post_media_image" />
 
     <com.google.android.material.textfield.TextInputLayout
         style="@style/TuskyTextInput"
+
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:layout_weight="1"
         android:layout_marginStart="?dialogPreferredPadding"
-        android:layout_marginTop="?dialogPreferredPadding"
         android:layout_marginEnd="?dialogPreferredPadding"
-        app:hintEnabled="false"
+        android:layout_marginTop="?dialogPreferredPadding"
         app:counterEnabled="false"
-        app:counterTextColor="?android:textColorTertiary">
+        app:counterTextColor="?android:textColorTertiary"
+        app:hintEnabled="false">
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/imageDescriptionText"
-            android:inputType="textCapSentences|textMultiLine|textAutoCorrect"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:hint="@string/hint_description"
             android:gravity="start"
-            android:importantForAutofill="no" />
+            android:hint="@string/hint_description"
```

Comment:
this is not necessary since it is overridden from code anyway and the string is used only here it can be deleted as well
```suggestion
            tools:hint="Description"
```

Reply:
Done.

(It's a warning now: "don't use hard-coded strings" :-)

Reply:
False positive, if it starts with `tools` it is only for displaying the layout in Android Studio and won't end up in the apk.